apps/server/src/core/error/utils/error.utils.ts

```diff
@@ -20,4 +20,16 @@ export class ErrorUtils {
 	static isNestHttpException(error: unknown): error is HttpException {
 		return error instanceof HttpException;
 	}
+
+	static convertUnknownError(error: unknown, description?: string): HttpExceptionOptions {
```

Comment:
I think this should be named `createHttpExceptionOptions` or similar, because it does not simply convert an error but also has the description as second parameter.

Reply:
agree it can be renamed

Reply:
I renamed it to createHttpExceptionOptions but if we follow the argumentation that it should call something like
createHttpExceptionOptionsFromUnkownError?

Reply:
I don't think "FromUnknownError" is needed. I think it's even misleading, because you can also use it for known errors.